content/tutorials/tasks/use-xrpl-sidechains/set-up-iou-iou-bridge.md

```diff
@@ -0,0 +1,168 @@
+---
+html: set-up-iou-iou-bridge.html
+parent: use-xrpl-sidechains.html
+blurb: Steps to set up an IOU-IOU bridge.
+labels:
+  - Interoperability
+---
+# Set Up an IOU-IOU Bridge
```

Comment:
Oy. If this goes forward, seriously, we have to change the guidance on the use of the term _IOU_.